static/js/library/status.js

```diff
@@ -723,3 +723,14 @@ function update_result_status($child, status){
     $status_label.addClass(status).text(status);
 
 }
+
+function go_similar(event, elem, tmdbid, title){
+    if (tmdbid == ""){
+        return false;
+    }
+
+    $.redirect(url_base+'/sugg_similar', {
+           "tmdbid": tmdbid,
+	   "title": title
+    });
+}
```

Comment:
Rather than include jquery.redirect for just this one call you could have it send the browser to `/sugg_similar?tmdbid=987654` then when the ajax method is called on page load it can parse the tmdbid from the query string if there is one.